package.json

```diff
@@ -12,7 +15,8 @@
         "build-storybook": "build-storybook",
         "prepare": "husky install",
         "lint": "prettier --check .",
-        "format": "prettier --write ."
+        "format": "prettier --write .",
+        "semantic-release": "semantic-release"
```

Comment:
no-one should be running this locally, technically, and in your github actions you are using npx. 
I _think_ you can remove this line, but not 100% sure. If you can, definitely remove it, if not, this PR looks great. 
Great job with gihtub actions